.travis.yml

```diff
@@ -3,14 +3,21 @@ dist: xenial
 language: python
 
 python:
+  - '3.4'
+  - '3.5'
+  - '3.6'
   - '3.7'
+  - 'pypy3.5'
+  - '3.8-dev'
```

Comment:
Nice. However, we really don't need to have `3.8-dev`, and specially the `nightly` one, IHMO ;)

Reply:
Why not?

Reply:
I don't think there's any benefit IHMO. `dev` can change relatively a lot. I don't think there's any value in having it here honestly (unless we were married to some internal or prototype feature of it).

Anyway, I wouldn't totally be against this, so I will leave others give their opinion ;)

Reply:
Python 3.8 is almost there, so it makes sense to have tests for it.

In addition, I believe that it's better to know about any incompatibilities sooner rather than later.
Especially if running the tests costs nothing.

Reply:
It does make builds slower. But I have no strong opinion on `dev` either. I also believe that `nightly` is too much, things can still change there so if we fix an error in `nightly` and then `nightly` changes to make the fix unnecessary, we wasted time.

Reply:
Hi @carlosalberto and @Oberon00,
I removed `nightly` for now.